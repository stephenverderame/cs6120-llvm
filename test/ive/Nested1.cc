// NOLINTBEGIN
#include <cstdio>
int main()
{
    int m = 20;
    for (int i = 0; i < 10; ++i) {
        int p = i * 10 + m;
        for (int j = 0; j < i; ++j) {
            int k = j * i + p;
            printf("%d\n", k);
        }
    }
}
// NOLINTEND