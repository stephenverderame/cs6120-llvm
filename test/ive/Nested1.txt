30
40
42
50
53
56
60
64
68
72
70
75
80
85
90
80
86
92
98
104
110
90
97
104
111
118
125
132
100
108
116
124
132
140
148
156
110
119
128
137
146
155
164
173
182
