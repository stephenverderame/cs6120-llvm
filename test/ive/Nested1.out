Basic IV:   %10 = add nuw nsw i32 %9, 10
Basic IV:   %14 = add nuw nsw i32 %9, 1
Basic IV:   %9 = phi i32 [ %14, %8 ], [ 0, %2 ]
Basic IV:   %3 = phi i32 [ 0, %0 ], [ %6, %5 ]
Basic IV:   %6 = add nuw nsw i32 %3, 1
